src/app/src/main/kotlin/org/imperial/mrc/hint/AppProperties.kt

```diff
@@ -12,10 +12,10 @@ interface AppProperties {
 }
 
 @Configuration
-open class ConfiguredAppProperties: AppProperties {
+open class ConfiguredAppProperties(props: Properties = properties): AppProperties {
```

Comment:
This is kind of an ugly way to make this testable by allowing props other than those immediately read from disk by the companion object to be used.

Reply:
I can't think of a better way either and you're right it's good to test.